Remove obsolete mediaControlsBase and mediaControlsLocalizedStrings files

blino opened the pull request:
#### 3ce0e112b53ff30b29916fb41a1d65ef48ac1975
<pre>
Remove obsolete mediaControlsBase and mediaControlsLocalizedStrings files
<a href="https://bugs.webkit.org/show_bug.cgi?id=262034">https://bugs.webkit.org/show_bug.cgi?id=262034</a>

Reviewed by Michael Catanzaro, Fujii Hironori and Ross Kirsling.

The last usage of mediaControlsBaseUserAgentStyleSheet has been
removed in 221018@main for the GLib ports.

The last usage of mediaControlsLocalizedStringsJavaScript and
mediaControlsBaseJavaScript has been removed in 221364@main for the
GLib ports.

The last usage of mediaControlsLocalizedStrings has been removed in
235609@main for the Mac ports, and 264642@main for the Win port.

* Source/WebCore/Modules/mediacontrols/mediaControlsBase.css: Removed.
* Source/WebCore/Modules/mediacontrols/mediaControlsBase.js: Removed.
* Source/WebCore/PlatformPlayStation.cmake:
* Source/WebCore/en.lproj/mediaControlsLocalizedStrings.js: Removed.
* Tools/gtk/manifest.txt.in:
* Tools/wpe/manifest.txt.in:

Canonical link: <a href="https://commits.webkit.org/268575@main">https://commits.webkit.org/268575@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/4d899f0d053232193e3d40f00a795d39494361c9

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/19616 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/20037 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/20648 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/21509 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/18348 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/19853 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/23300 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/20179 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/19950 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/19834 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/19850 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/17055 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/22366 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/17034 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/17846 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/24151 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/18098 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/18020 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/22127 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/18628 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/15791 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/17775 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/4813 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/22130 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/18454 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rkirsling reviewed (approved):
Thanks for checking, I believe this should be fine.

blino committed:
Remove obsolete mediaControlsBase and mediaControlsLocalizedStrings files
https://bugs.webkit.org/show_bug.cgi?id=262034

Reviewed by Michael Catanzaro, Fujii Hironori and Ross Kirsling.

The last usage of mediaControlsBaseUserAgentStyleSheet has been
removed in 221018@main for the GLib ports.

The last usage of mediaControlsLocalizedStringsJavaScript and
mediaControlsBaseJavaScript has been removed in 221364@main for the
GLib ports.

The last usage of mediaControlsLocalizedStrings has been removed in
235609@main for the Mac ports, and 264642@main for the Win port.

* Source/WebCore/Modules/mediacontrols/mediaControlsBase.css: Removed.
* Source/WebCore/Modules/mediacontrols/mediaControlsBase.js: Removed.
* Source/WebCore/PlatformPlayStation.cmake:
* Source/WebCore/en.lproj/mediaControlsLocalizedStrings.js: Removed.
* Tools/gtk/manifest.txt.in:
* Tools/wpe/manifest.txt.in:

Canonical link: https://commits.webkit.org/268575@main
